web-client/iron-remote-gui/README.md

```diff
@@ -52,12 +52,31 @@ You can add some parameters for default initialization on the component `<iron-r
 
 ## `UserInteractionService` methods
 
-> `connect(username: string, password: string, host: string, authtoken: string): Observable<NewSessionInfo>`
+```typesccript
+ private connect(
+        username: string,
+        password: string,
+        destination: string,
+        proxyAddress: string,
+        serverDomain: string,
+        authToken: string,
+        desktopSize?: DesktopSize,
+        preConnectionBlob?: string,
+        kdc_proxy_url?: string,
+    ): Observable<NewSessionInfo> 
+```
 >
 > `username` and `password` are the credentials to use on the remote host.
+
 > `host` refers to the Devolutions Gateway hostname and port.
```

Comment:
This is out of date (named `destination` above)

Reply:
updated